src/util/validation.h

```diff
@@ -6,9 +6,9 @@
 #ifndef BITCOIN_UTIL_VALIDATION_H
 #define BITCOIN_UTIL_VALIDATION_H
 
-#include <string>
+#include <consensus/validation.h>
 
-class ValidationState;
```

Comment:
In commit "Format CValidationState properly in all cases" (ee4d66a38c5c09161f8a3e63047db6b6908687f4)

Replacing forward declaration with a full include seems unnecessary here, just making the diff bigger and perhaps unnecessarily slowing down compilation.